viewer/viewer-awt/src/main/java/org/icepdf/ri/common/utility/search/SearchPanel.java

```diff
@@ -108,16 +104,7 @@
     private JButton searchButton;
     // clear search
     private JButton clearSearchButton;
-    private DropDownButton filterDropDownButton;
-    private JCheckBoxMenuItem caseSensitiveCheckbox;
-    private JCheckBoxMenuItem wholeWordCheckbox;
-    private JCheckBoxMenuItem regexCheckbox;
-    private JCheckBoxMenuItem cumulativeCheckbox;
-    private JCheckBoxMenuItem showPagesCheckbox;
-    private JCheckBoxMenuItem commentsCheckbox;
-    private JCheckBoxMenuItem textCheckbox;
-    private JCheckBoxMenuItem outlinesCheckbox;
-    private JCheckBoxMenuItem destinationsCheckbox;
+    private SearchFilterButtonWrapper wrapper;
```

Comment:
Would really like to keep with some of the naming conventions in the code base.  Suggest 'wrapper' become 'searchFilterButton' to make more clear to other developers.